lib/api/api_auth.go

```diff
@@ -27,15 +27,25 @@ const (
 	randomTokenLength  = 64
 )
 
-func emitLoginAttempt(success bool, username, address string, evLogger events.Logger) {
+func emitLoginAttempt(success bool, username string, r *http.Request, evLogger events.Logger) {
 	evLogger.Log(events.LoginAttempt, map[string]interface{}{
 		"success":       success,
 		"username":      username,
-		"remoteAddress": address,
+		"remoteAddress": r.RemoteAddr,
```

Comment:
I'd like to pass the "real" IP to the events as well, but there's a risk in doing so. Anyone can set the `X-Forwarded-For` header, so we would have to at least restrict which IPs we trust.

Limiting this to the localhost IP range is the safest choice, but already falls flat once containers are involved. Maybe it's ok if we also allow IPs in the LAN range?